src/flownet/parameters/_relative_permeability.py

```diff
@@ -12,96 +10,145 @@
 from .probability_distributions import ProbabilityDistribution
 from ._base_parameter import Parameter, parameter_probability_distribution_class
 
-MAX_WORKERS = 5  # Number of workers to be allowed for permeability calculations.
 
-
-def gen_wog(parameters: pd.DataFrame, fast_pyscal: bool = False) -> WaterOilGas:
+def swof_from_parameters(param: Dict) -> str:
     """
-    Creates a PyScal WaterOilGas object based on the input parameters supplied.
+    Creates a SWOF table based on a dictionary of input parameters/values
 
     Args:
-        parameters: A dataframe consisting of all specified parameters.
-        fast_pyscal: Run pyscal in fast-mode skipping checks. Useful for large models/ensemble.
+        param (Dict): Dictionary of saturation and relative permeability endpoints
```

Comment:
No need to write the type in the docstring (we'll get it from the type defined in Python).

```suggestion
        parameters: Dictionary of saturation and relative permeability endpoints
```